packages/babel-parser/test/fixtures/experimental/numeric-separator/invalid-lol/input.js

```diff
@@ -0,0 +1 @@
+00_0
```

Comment:
I know that you copied "lol" from test262, but please call it `invalid-legacy-octal-literal`. It took two minutes to me to understand what lol was :laughing:

Reply:
Aww i like lol more 🤣but's a reasonably change... committing in a minute! 👌